src/interp_func.jl

```diff
@@ -59,3 +59,15 @@ function InterpolationData(id::InterpolationData, f)
         id.differential_vars,
         id.sensitivitymode)
 end
+
+# strip interpolation of function information
+function strip_interpolation(id::InterpolationData)
```

Comment:
This needs to extend not shadow from SciMLBase, so make sure to import.

Reply:
is `SciMLBase.strip_interpolation ... ` fine, or should I do an explicit `import SciMLBase: strip_interpolation

Reply:
namespacing is fine.